block/block-manager.md

```diff
@@ -2,13 +2,45 @@
 
 ## Abstract
 
-The block manager is a key component of full nodes and is responsible for block production or block syncing depending on the node type. Block syncing in this context includes retrieving the published blocks from the network (p2p network or DA network), validating them to raise fraud proofs upon validation failure, updating the state, and storing the validated blocks. A full node invokes multiple block manager functionalities in parallel, such as:
-
-* block production (only for sequencer full nodes)
-* block publication to DA network
-* block retrieval from DA network
-* block retrieval from Blockstore (which retrieves blocks from the p2p network)
-* block syncing
+The block manager is a key component of full nodes and is responsible for block production or block syncing depending on the node type. Block syncing in this context includes retrieving the published blocks from the network (P2P network or DA network), validating them to raise fraud proofs upon validation failure, updating the state, and storing the validated blocks. A full node invokes multiple block manager functionalities in parallel, such as:
```

Comment:
suggestion: `depending on the node type: sequencer or non-sequencer`

Reply:
applied